Makefile

```diff
@@ -45,6 +45,8 @@ SHELL:=/bin/bash
 	    rm -f nx-X11/config/cf/nxversion.def; \
 	    rm -f bin/nxagent; \
 	    rm -f bin/nxproxy; \
+	    rm -f nx*/install-sh; \
```

Comment:
NAK, this PR is bogus.

We should:
  - be fixing `configure.in`:
    - rename it to `configure.ac`
    - add proper automake dependencies to it
  - use `autoreconf -fvi` as part of roll-tarball.sh to generate the build system for the tarballs (which should automatically install install-sh and mkinstalldirs as symlinks if they don't exist and autoreconf determined that the build system is using automake.)